### DISCUSSION TITLE
Cache Control for Cloudflare Workers

### DISCUSSION THREAD

**Discussion by oleghalin** [nitrojs/nitro#919] [Q&A]
I see that its commented now in preset handler. I tried to use routeRules to set custom headers, but it doesnt affected on Cloudflare Workers, but it works on Node env.

**Comment by TheAlexLichter** [comment]
Feel free to raise an issue with a reproduction 👍🏻

**Reply by oleghalin** [reply]
Since Cloudflare Worker preset doesnt serve assets from runtime and it serves from KV storage directly (https://github.com/unjs/nitro/blob/main/src/runtime/entries/cloudflare.ts#L16) routeRules doesnt affect it all. I will be happy to share repro but it just require you to deploy this nuxt setup to cloudflare workers. 

https://stackblitz.com/edit/github-5u37it?file=nuxt.config.ts

**Comment by oleghalin** [comment]
I just opened PR which resolves this issue https://github.com/unjs/nitro/pull/922
